CHANGELOG/CHANGELOG-0.2.md

```diff
@@ -0,0 +1,44 @@
+## Release Summary
+
+This release focuses on the stabilization of Arktos as well as new features in multi-tenancy, scalability and unified VM/Container. Major improvements include: 
+
+* Multi-tenancy: multi-tenancy access control and short path support.
+* Scalability: API server data partitioning and active-active controller framework.  
```

Comment:
Scalability can be update to the following (as we added 130 changelog)

Scalability: API server data partitioning and performance test in AWS